db/migrations/052_new_roles.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "sequel/all_or_none_constraint"
+require "sequel/null_or_present_constraint"
+require "sequel/unambiguous_constraint"
```

Comment:
@rgalanakis Don't think you mean to add these here.

Reply:
Ah no good catch.